### DISCUSSION TITLE
X6项目打包后发布运行拖拽卡顿

### DISCUSSION THREAD

**Discussion by Alexandermclean** [antvis/X6#1710] [Q&A]
X6项目本地start选中9个互相连接的节点拖动，很顺畅不卡顿；为何build打包完发布，在测试环境选中同样的节点拖拽非常卡顿，附上卡顿时performance截图
![image](https://user-images.githubusercontent.com/14287592/147758556-4bd73e8b-da4a-4181-92d4-e47a531d6970.png)

**Comment by NewByVector** [comment]
本地测试了下，并没有这么大的反差，有 demo 看下吗？
